reports/Google-2018-report.md

```diff
@@ -65,5 +71,5 @@ Four months ago, as an electrical engineer, a community development job might ha
 <br>
 
 Patrick J. Park
-
+[ADD CONTACT DETAILS HERE]
```

Comment:
Add details on how to contact you and maybe what your current position is.

Patrick J Park
Senior undergraduate student in [Degree] at [University]
email address
githubid

Reply:
added 👍